jetty-ee9/jetty-ee9-servlets/src/test/java/org/eclipse/jetty/ee9/servlets/AbstractGzipTest.java

```diff
@@ -45,7 +45,7 @@ public abstract class AbstractGzipTest
 {
     protected static final int DEFAULT_OUTPUT_BUFFER_SIZE = new HttpConfiguration().getOutputBufferSize();
 
-    protected WorkDir workDir;
+    public WorkDir workDir;
```

Comment:
Necessary to run GzipDefaultServletTest in debug on IDE. (junit5 is more strict with setAccessible behaviors now)